tests/console/sssd_389ds_functional.pm

```diff
@@ -32,7 +32,7 @@ sub run {
 
     my $docker = "podman";
     if (is_sle) {
-        $docker = "docker";
+        $docker = "docker" if is_sle("<15-SP5");
```

Comment:
So for older version it will be what ?

Reply:
docker still works for older version, newer will use podman. Test is already engineered to use either container runtime (see VRs)

Reply:
Ah ok, didn't see the $docker above.